insights/tools/insights_inspect.py

```diff
@@ -25,9 +25,9 @@
 
 >>> insights-inspect insights.combiners.hostname.hostname
 IPython Console Usage Info:
-Enter 'make_pass.' and tab to get a list of properties
+Enter 'Hostname.' and tab to get a list of properties
 Example:
-In [1]: make_pass.<property_name>
+In [1]: Hostname.<property_name>
```

Comment:
This should actually be ...
Enter 'hostname.'
Line 40 below should also be lowercase 'hostname'

Reply:
ditto

Reply:
Updated